src/components/interface/MellowLpDepositForm/components/LPMellowVaultDepositWindow.tsx

```diff
@@ -78,15 +84,15 @@ const LPMellowVaultDepositWindow: React.FunctionComponent<LPMellowVaultDepositWi
       <Typography
         variant="body1"
         sx={{
-          fontSize: '16px',
+          fontFamily: 'DM Sans',
+          fontSize: '14px',
           lineHeight: '1.6',
-          color: '#E5E1F9',
-          marginTop: '8px',
+          color: '#e5e1f9d9',
+          marginTop: '16px',
```

Comment:
```suggestion
          marginTop: (theme) => theme.spacing(4),
```

Reply:
fixed in a880d81b18fc358da6db21e427119ac7f03e85ef